### ISSUE TITLE
Still finished serach with no seeds

### ISSUE THREAD

**Issue by 8bitFra** [KaptainWutax/SeedCracker#184]
Anyway I knew this thing about 1.16 chunks. To be sure I tried very far from spawn (more than 100k) to be 100% sure I was in new chunks but nothing, same error! (Finished dungeon serach with no seeds.)

_Originally posted by @8bitFra in https://github.com/KaptainWutax/SeedCracker/issues/163#issuecomment-786578829_

**Comment by bat1010101** [comment]
same here

**Comment by MrAppu** [comment]
Try going to the end

**Comment by MrAppu** [comment]
Did you test it in another servers?

**Comment by 8bitFra** [comment]
Yes, if i host clean paper server it works. The problems start on some online servers, excluding those that you see actively blocking seed cracking I didn't understand why in some servers (like this one https://github.com/KaptainWutax/SeedCracker/issues/163#issuecomment-772665868) I can't find the seed.

**Comment by MrAppu** [comment]
I think bungeecord plugin is messing with it. I had some issues with it. Can you try it?

**Comment by 8bitFra** [comment]
I tried with my local Travertine server and **without plugins** it works fine (obv on the latest travertine & paper builds). Do you know the name of any bungeecord plugin that blocks seed cracking?

**Comment by 8bitFra** [comment]
Maybe I understand why it doesn't work. I'm not 100% sure but I think it's due to some plugins (like orebfuscator) or the native paper option to prevent x-ray messing with the mod. Do you know how I can bypass these 2 things?

**Comment by MrAppu** [comment]
Sorry man, I dont know how to bypass it

**Comment by 8bitFra** [comment]
Okay, I figured out how to partially bypass this. Looking at already closed threads (https://github.com/KaptainWutax/SeedCracker/issues/69, https://github.com/KaptainWutax/SeedCracker/issues/58) I was able to get it to work with paper anti x-ray on both engines. Unfortunately if orebfuscator is installed the mod still doesn't work. I don't know at this point if there is a solution for this, if anyone knows what I'm doing wrong let me know. Thank you.

**Comment by MrAppu** [comment]
All the best man

**Comment by mayureshpandharikar** [comment]
Its collected more than enough seeds ans still won't give me seed. I treid this in small online cross platform server 4b4t.live . My brother used it and it worked perfectly. Dungeons wont show up in xray and i think other stuff might hindering so i wanred to do only dungeon. Can you help?

**Comment by MrAppu** [comment]
Did you enable the dungeon option

**Comment by 8bitFra** [comment]
> Its collected more than enough seeds ans still won't give me seed. I treid this in small online cross platform server 4b4t.live . My brother used it and it worked perfectly. Dungeons wont show up in xray and i think other stuff might hindering so i wanred to do only dungeon. Can you help?

Have you seen the solutions proposed by the threads I mentioned above?

**Comment by zykrahgaming** [comment]
I modified the mod in a way that makes using it on anti-xray enabled servers a lot easier: https://github.com/zykrahgaming/SeedCracker/releases/tag/v0.2.4-beta-modified
(It just disables emerald ore finder and enables dungeon finder by default)

**Comment by 8bitFra** [comment]
> I modified the mod in a way that makes using it on anti-xray enabled servers a lot easier: https://github.com/zykrahgaming/SeedCracker/releases/tag/v0.2.4-beta-modified
> (It just disables emerald ore finder and enables dungeon finder by default)

I don't think this solves the problem. This plugin is very aggressive in hiding underground blocks (in fact it also causes a lot of lag in weaker clients) because it also hides those exposed to the air when the player is not nearby. So the only thing I can think of to get the seed of these worlds is to have a version of the mod where you enter the dungeon / biome fields manually. I'm not sure if this is actually the only solution, possibly if someone is more experienced than me let me know.

**Comment by Daynios** [comment]
i have now relaunched the 4b4t.live server, have a check now, it should work. if not, contact me on discord- daynios#0364

**Comment by mayureshpandharikar** [comment]
It worked! Thank you.
Is there any application on Android in which i can get enchantment table
seed or world seed?
